lxd/images.go

```diff
@@ -1017,6 +1018,8 @@ func imagesPost(d *Daemon, r *http.Request) response.Response {
 			return errors.Wrapf(err, "Failed syncing image between nodes")
 		}
 
+		d.State().Events.SendLifecycle(projectName, lifecycle.ImageCreated.Event(info.Fingerprint, projectName, op.Requestor(), log.Ctx{"type": info.Type}))
```

Comment:
We should make sure that when LXD automatically downloads an image (as happens during `lxc launch`) that we also issue this event upon initial image download.

Reply:
Just added that event in `ImageDownload` under `lxd/daemon_images.go`